docs/modules/integrations/pages/azure-devops-exporter.adoc

```diff
@@ -47,6 +47,25 @@ Please make sure that the `bdd.configuration.formats` property includes JSON val
 |`AUTOMATED` or `MANUAL`
 |The scenario part to export into `Steps` section of `Test Case` work item, the remaining scenario part is exported into `Summary` section.
 
+|`azure-devops-exporter.test-run.name`
+|
+|string
+a|The test run name.
+
+Setting of this property leads to creation of a new test run with the specified name.
+
+[IMPORTANT]
+====
+* All Azure DevOps Test Cases specified by the `testCaseId` meta tags on VIVIDUS scenarios within current VIVIDUS test run should belong the the Azure DevOps Test Plan pointed by the `azure-devops-exporter.test-run.plan-id` property.
+* Its not allowed to mix `manual` (contains only comments) and `automated` (contains at least one actionable step) test cases within one Azure DevOps Test Run.
+* The exporter mechanics do not support having the same Azure DevOps Test Case to be included into several Azure DevOps Test Suites within the specified Azure DevOps Test Plan.
+====
+
+|`azure-devops-exporter.test-run.plan-id`
+|
+|number
+|The ID of test plan that the test run belongs to. This property is mandatory if the `azure-devops-exporter.test-run.name` property is specified. The plan id can be found by opening appropriate Azure DevOps Test Plan in "Test Plans" and copying number value of the `planId` query parameter in the page URL.
```

Comment:
the role if this property is not clear

Reply:
the role of this property is too technical for QAs since it's used to find test points that are "invisible", but since it's required to link test cases to test suites within test plan it won't be an issue for them to find this id

Reply:
added